### DISCUSSION TITLE
Group Habits

### DISCUSSION THREAD

**Discussion by TaakoMagnusen** [iSoron/uhabits#723] [Feature Requests]
For me the reason i want to track some habits is to make sure i'm getting a well rounded set of activities in, we should be able to make groups of habits and check off stuff inside it that helps check off the category.

Example: Exercise group contains weightlifting and cardio. I want to so **either** of them to check off exercise for the day.

**Comment by hiqua** [comment]
Duplicate of https://github.com/iSoron/uhabits/discussions/495
